app/controllers/reviews_controller.rb

```diff
@@ -54,4 +57,12 @@ def destroy
   def review_params
     params.require(:review).permit(:description, :rating, :user, :bourbon)
   end
+
+  def authorize_user!
+    user = Review.find(params[:id]).user
+    unless current_user == user
+      flash[:alert] = "You Are Not Authorized To View The Page"
+      redirect_to after_sign_in_path_for(current_user)
+    end
```

Comment:
Nice job with these methods in general it's good practice to keep skinny controllers and since some of the logic is also repetitive on this and the Bourbon controller there could be a way to DRY up this code by putting the method on the application controller which these inherit from.